Source/JavaScriptCore/runtime/ProgramExecutable.cpp

```diff
@@ -145,7 +145,7 @@ JSObject* ProgramExecutable::initializeGlobalProperties(VM& vm, JSGlobalObject*
                 bool hasProperty = globalLexicalEnvironment->hasProperty(globalObject, entry.key.get());
                 RETURN_IF_EXCEPTION(throwScope, nullptr);
                 if (hasProperty)
-                    return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '", String(entry.key.get()), "'"));
+                    return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '"_s, String(entry.key.get()), '\''));
```

Comment:
Should be able to pass `entry.key` without `String` to avoid reference count churn.